cogs5e/sheetManager.py

```diff
@@ -852,20 +852,31 @@ def parse_args(self, args):
                 else:
                     character['settings']['critdmg'] = critdmg
                     out += "\u2705 Critdmg set to {}.\n".format(critdmg)
-            if arg == 'hocrit':
-                hocrit = list_get(index + 1, None, args)
-                if hocrit is None:
-                    out += '\u2139 Half-orc crits are currently {}.\n' \
-                        .format("enabled" if character['settings'].get('hocrit') else "disabled")
+            if arg == 'critdice':
+                critdice = list_get(index + 1, None, args)
+
+                if critdice is None:
+                    character['settings']['critdice'] += int(character['settings']['hocrit'] == True)
+                    character['settings']['hocrit'] = False
+                    out += '\u2139 Extra crit dice are currently set to {}. Use "!csettings critdice reset" to reset it.\n' \
+                        .format(str(character['settings'].get('critdice')) if character['settings'].get(
+                        'critdice') is not '0' else "0")
+                elif critdice.lower() == 'reset':
+                    character['settings']['critdice'] = 0
+                    character['settings']['hocrit'] = False
+                    out += "\u2705 Extra crit dice reset.\n"
                 else:
                     try:
-                        hocrit = get_positivity(hocrit)
-                    except AttributeError:
-                        out += '\u274c Invalid input. Use "!csettings hocrit false" to reset it.\n'
+                        critdice = int(critdice)
+                    except (ValueError, TypeError):
+                        out += '\u274c Invalid number. Use "!csettings critdice reset" to reset it.\n'
                     else:
-                        character['settings']['hocrit'] = hocrit
-                        out += "\u2705 Half-orc crits {}.\n".format(
-                            "enabled" if character['settings'].get('hocrit') else "disabled")
+                        if not 1 <= critdice <= 20:
+                            out += '\u274c Extra crit dice must be between 1 and 20.\n'
```

Comment:
Probably add text explaining how to reset? I can imagine users going `!csettings critdice 0`